framework/src/wifi_manager/wpa_wifi_utils.c

```diff
@@ -309,13 +309,18 @@ wifi_utils_result_e wifi_utils_deinit(void)
 wifi_utils_result_e wifi_utils_scan_ap(void *arg)
 {
 	wifi_utils_result_e wuret = WIFI_UTILS_FAIL;
-	int8_t ret = WiFiScanNetwork();
+	int8_t ret = WiFiRegisterScanCallback(&wifi_scan_result_callback);
+	if (ret != SLSI_STATUS_SUCCESS) {
+		ndbg("[WU] [ERR] Register Scan Callback(%d)\n", ret);
+		return wuret;
+	} 
+	ndbg("[WU] Register Scan Callback(%d)\n.", ret);
```

Comment:
"\n." -> ".\n"

Reply:
Sorry for mistake! Modification completed.